src/main/java/com/jcabi/github/mock/MkPull.java

```diff
@@ -212,47 +230,58 @@ public JsonObject json() throws IOException {
         final String branch = xml.xpath("base/text()").get(0);
         final String head = xml.xpath("head/text()").get(0);
         final String[] parts = head.split(MkPull.USER_BRANCH_SEP, 2);
-        final List<String> patches = xml.xpath("patch/text()");
-        final String patch;
-        if (patches.isEmpty()) {
-            patch = "";
-        } else {
-            patch = patches.get(0);
+        final JsonObject obj = new JsonNode(xml).json();
```

Comment:
@paulodamaso Well, the way I understand it, ``MkPull.json()`` should return the Json representation of itself. Since MkPull is originally represented in XML, why don't we simply return this ``obj``? Why do we make this method extract only specific nodes from the Json? :D See [my](https://github.com/jcabi/jcabi-github/issues/1424#issuecomment-414566747) comment. 

Know what I mean? I think this method should look like this:

```java
@Override
public JsonObject json() throws IOException {
    return new JsonNode(xml).json(
        this.storage.xml().nodes(this.xpath()).get(0)
    );
}
```